ui/src/app/views/resource-card/component.tsx

```diff
@@ -36,7 +36,7 @@ const ResourceCardMenu: React.FunctionComponent<{
         ['notebook', () => {
           return <React.Fragment>
             <MenuItem icon='pencil' onClick={onRenameNotebook}>Rename</MenuItem>
-            <MenuItem icon='copy' onClick={onCloneResource}>Clone</MenuItem>
+            <MenuItem icon='copy' onClick={onCloneResource}>Duplicate</MenuItem>
```

Comment:
should we change the names of the methods too? i.e. onCloneResource -> onDuplicateResource.  Just to have no lingering confusion

Reply:
So I think we discussed this, we're leaving the api as clone (as this mirrors what FC has), and I think we decided the "line" here was what actually appears in the UI, everything else should stay the same

Reply:
okay in that case 👍